dockerfiles/ink-ci-linux/Dockerfile

```diff
@@ -35,12 +38,18 @@ RUN	set -eux; \
 # Installs the latest common nightly for the listed components,
 # adds those components, wasm target and sets the profile to minimal
 	rustup toolchain install nightly --target wasm32-unknown-unknown \
-			--profile minimal --component rustfmt clippy miri rust-src; \
+		--profile minimal --component rustfmt clippy miri rust-src; \
 	rustup default nightly; \
 # We require `xargo` so that `miri` runs properly
 # We require `grcov` for coverage reporting and `rust-covfix` to improve it.
-	cargo install grcov rust-covfix cargo-contract xargo; \
+	cargo install grcov rust-covfix xargo; \
+# download the latest cargo-contracts binary
+	curl -L "https://gitlab.parity.io/parity/cargo-contract/-/jobs/artifacts/master/raw/artifacts/cargo-contract/cargo-contract?job=build" \
```

Comment:
It should be the `cargo-contract` version from the tag [ink-ci-v0.8.0](https://github.com/paritytech/cargo-contract/releases/tag/ink-ci-v0.8.0). Is there a way to do this?